frontends/web/src/components/balance/balance.tsx

```diff
@@ -36,24 +37,29 @@ export const Balance: FunctionComponent<Props> = ({
       <header className={style.balance}></header>
     );
   }
+
+  // remove trailing zeroes from Bitcoin balance
```

Comment:
Why do you say "Bitcoin" balance? This component is used for all balances also ETH/tokens.

Reply:
done :+1: